markets/perps-market/storage.dump.sol

```diff
@@ -565,6 +565,25 @@ library GlobalPerpsMarketConfiguration {
     }
 }
 
+// @custom:artifact contracts/storage/LiquidationAmount.sol:LiquidationAmount
+library LiquidationAmount {
+    struct Data {
+        uint128 timestamp;
+        uint128 amount;
```

Comment:
the sizes of orders are all 128s and will be in native units so i dont think we'll ever go over, but could be convinced otherwise

Reply:
2^127-1= 1.7014118e+38
So we have this token, it's called shiba, it's in the 1000's of trillions, that's 12 zeroes, + the 18 decimals points, that's 30, leaves 8 zeros 🤔 
I mean we kind of need a check somewhere to avoid overflow

Reply:
all of the casting happens safely so if it can't cast, it will revert so they wont under/overflow, but it could brick the system if the market can't proceed with normal functions.  i'll make a note to add a test to see what happens.  can definitely up it to 256, i think it was 128 in v2?

Reply:
yeah sorry i just have recently started reading v2 and am not sure why we switched 256 to 128. But we definitely could find ourselves in a situation where we have a lot of digits to work with, especially for those meme tokens that sell for really low and have really large supply metrics